packages/xo-web/src/common/xo/index.js

```diff
@@ -3303,6 +3303,8 @@ export const deployProxyAppliance = (license, sr, { network, proxy, ...props } =
     ...props,
   })::tap(subscribeProxies.forceRefresh)
 
+export const connectProxyAppliance = proxyInfo => _call('proxy.register', proxyInfo)::tap(subscribeProxies.forceRefresh)
```

Comment:
[Then](https://github.com/vatesfr/xen-orchestra/pull/6556#discussion_r1033027195) let's keep it homogeneous and call it `registerProxyAppliance`.